specification/purview/data-plane/Azure.Analytics.Purview.DevopsPolicies/preview/2022-11-01-preview/purviewDevopsPolicy.json

```diff
@@ -0,0 +1,611 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DevopsSqlPerfMonitorPolicy_CreateOrReplace": {
+            "$ref": "./examples/DevopsSqlPerfMonitorPolicy_CreateOrReplace.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DevopsPolicy"
+        ],
+        "description": "Get devops sql perf monitor policy",
+        "operationId": "DevopsSqlPerfMonitorPolicy_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "purviewResourceName",
+            "description": "Name of resource as registered on the Azure purview portal",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DevopsPolicy"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DevopsSqlPerfMonitorPolicy_Get": {
+            "$ref": "./examples/DevopsSqlPerfMonitorPolicy_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DevopsPolicyDecisionRule": {
+      "type": "object",
+      "required": [
+        "effect",
+        "principals",
+        "principalGroups"
+      ],
+      "properties": {
+        "effect": {
+          "description": "The effect for rule",
+          "enum": [
+            "Permit"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Decision",
+            "modelAsString": true
+          }
+        },
+        "purviewRoleName": {
+          "type": "string",
+          "description": "This is field will indicate the role of a devops policy."
+        },
+        "principals": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Azure Active Directory Ids for users"
+          }
+        },
+        "principalGroups": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Azure Active Directory Ids for groups"
```

Comment:
As above, you should move this description up a level.

Reply:
Moved this description one level up. Also, added a description to item.